templates/base_index.html

```diff
@@ -5,9 +5,39 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
     <title itemprop="name">MAAS {% block page_title %}{% endblock %}</title>
-    {% with css_file="css/main.css" %}
-      {% include "includes/head.html" %}
-    {% endwith %}
+    {% include "includes/tag_manager.html" %}
+
+    <!-- javascript -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
```

Comment:
why do we need all this JS suddenly?

Reply:
It was always there. I moved single use include to a head to make it easier to maintain. I will create an issue to [investigate and potentially remove jQuery](https://github.com/canonical-web-and-design/maas.io/issues/653#issuecomment-1032448033).